modules/bclconvert/meta.yml

```
@@ -0,0 +1,45 @@
+name: "bclconvert"
+description: Demultiplex Illumina BCL files
+keywords:
+  - demultiplex
+tools:
+  - "bclconvert":
+      description: "Demultiplex Illumina BCL files"
+      homepage: "https://support.illumina.com/sequencing/sequencing_software/bcl-convert.html"
+      documentation: "https://support-docs.illumina.com/SW/BCL_Convert/Content/SW/FrontPages/BCL_Convert.htm"
+      tool_dev_url: "NA"
+      doi: "NA"
+      licence: "NA"
+
+input:
+  - samplesheet:
+      type: file
+      description: "Input samplesheet"
+      pattern: "*.{csv}"
+  - run_dir:
+      type: directory
+      description: "Input run directory"
```

```suggestion
      description: "Input run directory"
```

containing what? `bcl` files I would assume?

---

yep, bcl data and RunInfo.xml in the directory structure as the sequencer has written it